src/agent/src/watcher.rs

```diff
@@ -35,7 +35,9 @@ const MAX_SIZE_PER_WATCHABLE_MOUNT: u64 = 1024 * 1024;
 const WATCH_INTERVAL_SECS: u64 = 2;
 
 /// Destination path for tmpfs
-const WATCH_MOUNT_POINT_PATH: &str = "/run/kata-containers/shared/containers/watchable/";
+const WATCH_MOUNT_POINT_PATH: &str =
+    "/run/kata-containers/shared/containers/passthrough/watchable/";
+// FIXME: hardcode passthrough
```

Comment:
Should consider 3 types:

- golang runtime path
- runtime-rs passthrough path
- runtime-rs rafs path(not available yet)